src/xdocs/config_naming.xml

```diff
@@ -1261,8 +1261,26 @@ class MyClass {
          An example of how to configure the check is:
         </p>
         <source>
-&lt;module name="MemberName"/&gt;
+&lt;module name=&quot;MemberName&quot;/&gt;
         </source>
+        <p>Code Example:</p>
+        <pre>
+class MyClass {
+  public int num1; // OK
+  protected int num2; // OK
+  int num3; // OK
+  private int num4; // OK
+
+  public int NUM1; // violation, name 'NUM1'
+                   // must match pattern '^[a-z][a-zA-Z0-9]*$'
+  protected int NUM2; // violation, name 'NUM2'
+                      // must match pattern '^[a-z][a-zA-Z0-9]*$'
+  int NUM3; // violation, name 'NUM3'
+            // must match pattern '^[a-z][a-zA-Z0-9]*$'
+  private int NUM4; // violation, name 'NUM4'
+                    // must match pattern '^[a-z][a-zA-Z0-9]*$'
+}
+        </pre>
         <p>
           An example of how to configure the check for names that begin with
           <code>&quot;m&quot;</code>, followed by an upper case letter, and then letters and
```

Comment:
Description was not updated when `applyTo` were added.

Reply:
Done.